### DISCUSSION TITLE
PathbuilderDemo: unclear elements in Customizer

### DISCUSSION THREAD

**Discussion by 3Dietrich** [dinther/pathbuilder#4] [Q&A] [answered]
I see that **show_demo** and the code (with it's comments) seems to be coordinated with each other. But the 'example shapes'.. I don't know when they are used (well.. I could of course find it in the code, but you know.. this is not a "demo"). Maybe a short hint what shape is used in what demo?

**Comment by dinther** [comment, marked as answer]
I tidied up the demos and put them in separate files. Hopefully that makes the examples more accessible.

**Comment by 3Dietrich** [comment]
I do not see through it completely, but in any case it shows much more and clearer than before! Excellent work!

**Reply by dinther** [reply]
You are more than welcome to make suggestions Dietrich. I have been looking at this for so long that I can't be objective in what is clear and what is not.
